src/utils/Constants.ts

```diff
@@ -48,7 +48,7 @@ export class Constants {
 
     static readonly AUTHORIZATION_CODE_INDEX_NAME = "authCode-updated-index";
 
-    static readonly TOKEN_EXPIRY_SECONDS = 3600;
+    static readonly TOKEN_EXPIRY_SECONDS = 3.154e+9;
```

Comment:
What's this for?

Reply:
yeah....had added this for generating a long expiry token, forgot to revert it. i have reverted it now.